src/client.ts

```diff
@@ -1051,6 +1058,24 @@ export class AwaitConnection {
     this.connected = false;
   }
 
+  isClosed(): boolean {
+    return !this.connected;
+  }
+
+  public get user(): ConnectionUser | null {
+    return this._user;
+  }
+
+  setUser(user: ConnectionUser | null): void {
```

Comment:
In addition to the naming in https://github.com/edgedb/edgedb-js/pull/52/files#r441916382, I also think that this shouldn't be a "public" method. We can't make it "private", but I'd at least prefix it with an undersore and mark it `/** @internal */`.